Main/ClientApp/src/shared/components/DrawerAppBar.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from "react";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import LoginAppBar from "./LoginAppBar";
+import ClippedDrawer from "./ClippedDrawer";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: "flex"
+    },
+    content: {
+      flexGrow: 1,
+      padding: theme.spacing(3)
+    },
+    // necessary for content to be below app bar
+    toolbar: theme.mixins.toolbar
+  })
```

Comment:
Though the material.ui styling is easy to copy and apply to our website, it will hurt us in the long run as it is very custom. We might need to discuss what to do with styling/css